### PR TITLE
mixnode documentation update

### PR DESCRIPTION
**Pull request by Pawnflake**
# Description

Closes: #XXXX

Removed http and https ports from ufw rules in mixnode documentation

### CODE REVIEW

**Comment by mfahampshire** [comment]
@Pawnflake please retarget to next week's release (`v1.1.20`)

**Review by mfahampshire — APPROVED** [review]
lgtm

File: documentation/docs/src/nodes/mix-node-setup.md [hunk 1]
```diff
@@ -228,12 +228,14 @@ sudo ufw enable
...
-Finally open your mix node's p2p port, as well as ports for ssh, http, and https connections, and ports `8000` and `1790` for verloc and measurement pings:
+Finally open your mix node's p2p port, as well as ports for ssh and ports `8000` and `1790` for verloc and measurement pings:
 
 ```
-sudo ufw allow 1789,1790,8000,22,80,443/tcp
+sudo ufw allow 1789,1790,8000,22/tcp
 # check the status of the firewall
 sudo ufw status
+# restart firewall
+sudo sytstemctl restart ufw
```
Comment (twofaktor) [inline documentation/docs/src/nodes/mix-node-setup.md]: This is not needed

File: documentation/docs/src/nodes/mix-node-setup.md [hunk 2]
```diff
@@ -228,12 +228,14 @@ sudo ufw enable
...
 
-Finally open your mix node's p2p port, as well as ports for ssh, http, and https connections, and ports `8000` and `1790` for verloc and measurement pings:
+Finally open your mix node's p2p port, as well as ports for ssh and ports `8000` and `1790` for verloc and measurement pings:
 
 ```
-sudo ufw allow 1789,1790,8000,22,80,443/tcp
+sudo ufw allow 1789,1790,8000,22/tcp
 # check the status of the firewall
 sudo ufw status
+# restart firewall
```
Comment (twofaktor) [inline documentation/docs/src/nodes/mix-node-setup.md]: This is not needed

**Review by twofaktor — CHANGES_REQUESTED** [review]

File: documentation/docs/src/nodes/mix-node-setup.md (hunk 1 above)
Comment (mfahampshire) [reply documentation/docs/src/nodes/mix-node-setup.md]: agree @Pawnflake can you please remove these lines for me? thanks

Comment (twofaktor) [inline documentation/docs/src/nodes/mix-node-setup.md]:
```suggestion

```

File: documentation/docs/src/nodes/mix-node-setup.md (hunk 2 above)
Comment (twofaktor) [inline documentation/docs/src/nodes/mix-node-setup.md]:
```suggestion
```
```

File: documentation/docs/src/nodes/mix-node-setup.md [hunk 3]
```diff
@@ -228,12 +228,14 @@ sudo ufw enable
...
+sudo sytstemctl restart ufw
 ```
```
Comment (twofaktor) [inline documentation/docs/src/nodes/mix-node-setup.md:237]:
```suggestion

```

**Review by twofaktor — CHANGES_REQUESTED** [review]
Changes proposed!

**Pull request merged**